src/main/java/seedu/address/model/person/Expense.java

```diff
@@ -87,25 +87,19 @@ public int hashCode() {
     public String toString() {
         final StringBuilder builder = new StringBuilder();
 
-//        if (description.isEmpty()) {
```

Comment:
Maybe can use this for optional description @yuming7144

Reply:
This will be the displayed message when expense is created/modified. Description is already optional! Thanks for the suggestion though really!